formats/avro/src/main/java/io/cloudevents/avro/AvroSerializer.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.cloudevents.avro;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+import java.util.HashMap;
+
+import io.cloudevents.CloudEvent;
+import io.cloudevents.CloudEventData;
+import io.cloudevents.core.v1.CloudEventV1;
+import io.cloudevents.AvroCloudEvent;
+
+class AvroSerializer {
+
+    public static final AvroCloudEvent toAvro(CloudEvent e) {
```

Comment:
No need for final and public:
```suggestion
    static AvroCloudEvent toAvro(CloudEvent e) {
```